src/applications/resources-and-support/constants/index.js

```diff
@@ -0,0 +1 @@
+export const RESULTS_PER_PAGE = 10;
```

Comment:
We could possibly bind this to the app as a value under `defaultProps`. Not related to this work exactly but a general improvement.